res/TensorFlowLiteRecipes/Net_Squeeze_Squeeze_000/test.rule

```diff
@@ -0,0 +1,6 @@
+# To check if Squeeze is substituted to Reshape op
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "SQUEEZE_COUNT"           $(op_count SQUEEZE) '=' 0
+RULE    "RESHAPE_COUNT"           $(op_count RESHAPE) '=' 2
```

Comment:
For now, since `--all` option was removed, it would not be occurred but just for note.
If `remove_redundant_reshape` option is enabled in the related tests, the number of `Reshape` is 1.

Reply:
> since --all option was removed,

it's not removed yet :) This maybe revised then.

> If remove_redundant_reshape option is enabled

if there would be any cases, we may revsit this